fern/pages/docs/getting-started/docs-overview.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: Fern Docs Overview
+description: Learn about key features of Fern's Docs
+subtitle: A static site generator for developer-focused companies to build and host a beautiful, professional docs site.
+layout: overview
+---
+
+## Key Features
+
+<CardGroup cols={3}>
+  <Card 
+    title="Professional Design" 
+    icon="paintbrush" 
+    href="/learn/docs/building-your-docs/custom-css-js"
+  >
+    Match your docs to your brand with customizable colors, fonts, and styles.
+  </Card>
+
+  <Card 
+    title="Powered by OpenAPI" 
+    icon="rotate" 
+    href="/learn/docs/content/generate-api-ref"
+  >
+    Auto-generate API docs from OpenAPI and AsyncAPI specs.
+  </Card>
+
+  <Card 
+    title="Git Sync" 
+    icon="code" 
+    href="/learn/docs/content/write-markdown"
+  >
+    Version and manage docs with your Git provider using Markdown or MDX.
+  </Card>
+
+  <Card 
+    title="Interactive API Playground" 
+    icon="play" 
+    href="/learn/docs/content/api-playground"
+  >
+    Test APIs directly in the docs with auto-populated credentials.
+  </Card>
+
+  <Card 
+    title="SDK Snippets" 
+    icon="box" 
+    href="/learn/docs/config/sdk-snippets"
+  >
+    Example code auto-updates as your SDKs evolve.
+  </Card>
+
+  <Card 
+    title="Flexible Component Library" 
+    icon="puzzle-piece" 
+    href="/learn/docs/components/overview"
+  >
+    Use pre-built or custom React components for a polished look.
+  </Card>
+
+</CardGroup>
+
+## Documentation Types
+
+<Tabs>
+  <Tab title="For Developers">
```

Comment:
it was a bit hard to follow what you mean by Documentation Types -- whats the goal of this section?

Reply:
My goal here is to explain that fern docs support multiple types of content:
- landing page
- guides
- API ref
- changelog

Reply:
revised in commit [c5fe1f5](https://github.com/fern-api/fern/pull/5500/commits/c5fe1f53fb7310f5541a38f20780543987a2c01d)